apps/chrome.py

```diff
@@ -75,8 +75,8 @@ def jump_tab(m):
     'new tab': Key('cmd-t'),
     'close tab': Key('cmd-w'),
     '(reopen | unclose) tab': Key('cmd-shift-t'),
-    '(next tab | goneck)': Key('cmd-alt-right'),
-    '((last | prevous | preev) tab | gopreev)': Key('cmd-alt-left'),
+    '(next tab | goneck)': Key('cmd-shift-]'),
```

Comment:
I'm curious, do these not work for you? Both of them seem to work for me.

Reply:
Thanks for merging! They don't work for me possibly because of something else I've installed. The ones I changed them to are also more common for Mac apps generally.